includes/Admin/Import/ImportSermons.php

```diff
@@ -191,8 +199,12 @@ public function process_step() {
 				}
 
 				if ( $audio ) {
-					update_post_meta( $message_id, 'audio_url', $audio );
-					$item->update_meta_value( 'audio_url', $audio );
+          $audio_url = $audio;
+          if( $options['sideload_audio'] ) {
+            $audio_url = $this->sideload_media_and_get_url( $message_id, $audio );
```

Comment:
@jjroley this looks really good. Instead of setting the `$audio_url` here, try to sideload and make sure that we actually have a url before setting the variable. If the sideload failed, we should fall back to the original url.